Makefile

```diff
@@ -79,7 +81,11 @@ lint:
 presto-server-rpm.rpm:
 	wget 'https://repository.sonatype.org/service/local/artifact/maven/content?r=central-proxy&g=com.facebook.presto&a=presto-server-rpm&e=rpm&v=RELEASE' -O $@
 
-smoke: clean-test-all test-images
+smoke: clean-test-all test-images _smoke
+
+smoke-configurable-cluster: clean-test-all _smoke
```

Comment:
Hmm, I added that because I wanted to make sure the contents of the tmp directory were cleared. That directory contains the offline installed used with Docker but with a configurable cluster I can only use the online installer.

Maybe I can do a separate target for cleaning that directory?